backend/schema/corpora_schema.md

```diff
@@ -1,125 +1,71 @@
-
-The multiple forms of the dataset naturally mean that validation and curation split into multiple levels.
-
-First, there is "basic" validation and curation that is applied to the original form of the dataset. This checks that Corpora's
-[fundamental requirements](#Original-Dataset-Requirements) are met. These are things that are just absolutely needed for the system to work. For example,
-Corpora will not host PII even if a submitter really wants it to. Second, there is the curation of the integration-focused remix dataset. This adapts
-the dataset to all the requirements of the [Corpora remix schema](#Remix-Metadata).
-
-Corpora's curation policies will be very strict regarding the fundamental requirements. Datasets that do not satisfy them will be rejected. But the
-integration-focused curation will be more flexible. While curation is still driven by human interaction, we will strongly urge submitters to supply all
-the required information. And we will also urge them to change their own datasets to match the Corpora schema so that the difference between the "original"
-and the "remix" is very small. If submitters decline to do so, we'll talk with them about their reasons and record their feedback, but we will still add
-the dataset to Corpora, and we will still create the remix variant of the submitted data.
-
-
-### Submitter Incentives
-
-We recognize that Corpora cannot _make_ submitters do anything; following our schema is voluntary. But, Corpora can create incentives. For example, if a
-submitter doesn't want to include "tissue" then their dataset would not show up in tissue-related search results in the Portal. Or if a submitter uses a
-gene annotation that is incompatible with the Corpora schema annotation, then gene card information will not be available for their dataset in cellxgene.
-
-Over time we expect these incentives will lead to broader adoption of the Corpora schema and less friction around creating the remix datasets.
-
-
-### Software vs Policy
-
-Finally, we should draw a distinction between how we approach the schema and validation as a _policy_ versus how we implement it in _software_. The schema,
-especially anything outside of fundamental requirements, will change over time. It might change frequently and significantly as we get feedback from submitters
-and users. So while we might be strict about our validation policies at any given time, our software needs to be very flexible. We need to avoid a situation
-where we can't evolve schema and curation policies because it requires expensive software updates. There will be situations where software features or
-capabilities will be hard or impossible because we prioritize this flexibility.
+Then, when the submitter uploads the dataset to the Data Portal, the Data Portal will verify that the dataset does indeed follow an accepted version of the
+schema. If it does not, it will reject the dataset with an appropriate error message.
```

Comment:
Thinking ahead, would it be useful to let the user know that they can validate using CLI? Otherwise if something went wrong they'd need to wait for the upload to find out about it.

Reply:
That would be the intention until validation was available in the service. Also see the in-progress [Cellxgene Schema Guide](https://github.com/chanzuckerberg/cellxgene/blob/c15e0e99e74a3147bcce7c641816687e7afc087c/dev_docs/schema_guide.md) that would be part of that documentation.